yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -185,41 +210,71 @@ func parseFleetConfigFile(filePath string) ([]AssessMigrationDBConfig, error) {
 	}
 	defer file.Close()
 
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		line := scanner.Text()
-		if len(line) == 0 {
-			continue
-		}
-		dbConfig := parseFleetConfigLine(line)
-		dbConfigs = append(dbConfigs, dbConfig)
+	reader := csv.NewReader(file)
+	header, err := reader.Read()
+	if err != nil {
+		return nil, fmt.Errorf("failed to read fleet config file header: %w", err)
 	}
+	header = normalizeFleetConfFileHeader(header)
+
+	lineNum := 2
+	for {
+		record, err := reader.Read()
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			return nil, fmt.Errorf("failed to read line %d: %w", lineNum, err)
+		}
 
-	if err := scanner.Err(); err != nil {
-		return nil, err
+		dbConfig, err := createDBConfigFromRecord(record, header)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create config for line %d in fleet config file: %w", lineNum, err)
+		}
+		dbConfigs = append(dbConfigs, *dbConfig)
+		lineNum++
 	}
 
 	return dbConfigs, nil
 }
 
-/*
-Format: covers both SSL and non-SSL cases
+func createDBConfigFromRecord(record []string, header []string) (*AssessMigrationDBConfig, error) {
+	configMap := make(map[string]string)
+	for i, field := range header {
+		configMap[field] = strings.TrimSpace(record[i])
+	}
 
-	<dbtype>,<hostname>,<port>,<service_name>,<sid>,<tns_alias>,<username>,<password>,<schema>
-*/
-func parseFleetConfigLine(line string) AssessMigrationDBConfig {
-	config := strings.Split(line, ",")
-	return AssessMigrationDBConfig{
-		DbType:      config[0],
-		Host:        config[1],
-		Port:        config[2],
-		ServiceName: config[3],
-		SID:         config[4],
-		TnsAlias:    config[5],
-		User:        config[6],
-		Password:    config[7],
-		Schema:      config[8],
+	// Check if mandatory fields are present and non-empty
+	missingFields := []string{}
+	for _, field := range fleetConfRequiredFields {
+		if val, ok := configMap[field]; !ok || val == "" {
+			missingFields = append(missingFields, field)
+		}
 	}
+	hasDBIdentifier := false
```

Comment:
We should additionally validate that only ONE db identifier is passed (db_name, sid, tns_alias)

Reply:
would be good to validate here as well; but okay to take it up as an enhancement later.